lumino-explorer-api/src/main/java/org/rif/lumino/explorer/models/documents/LuminoHub.java

```diff
@@ -11,19 +14,37 @@
 
     private String url;
 
-    private Integer connectionCount;
+    private Set<String> connectedClients;
+
+    private long connectionCount;
+
+    private long maxConnectionsAllowed;
 
     public LuminoHub() {
+        this.connectedClients = new HashSet<>();
+        this.connectionCount = 0;
+        this.maxConnectionsAllowed = -1;
     }
 
     public LuminoHub(String url) {
         this.url = url;
+        this.connectedClients = new HashSet<>();
         this.connectionCount = 0;
+        this.maxConnectionsAllowed = -1;
     }
 
-    public LuminoHub(String url, int connectionCount) {
+    public LuminoHub(String url, HashSet<String> connectedClients) {
         this.url = url;
-        this.connectionCount = connectionCount;
+        this.connectedClients = connectedClients;
+        this.connectionCount = connectedClients.size();
+        this.maxConnectionsAllowed = -1;
```

Comment:
can we assign this to the constant INFINITE_CAPACITY?

Reply:
Good catch